.circleci/config.yml

```diff
@@ -350,8 +252,6 @@ jobs:
     working_directory: ~/repo
     steps:
       - setup_smoke_tests
-      - restore_cache:
```

Comment:
Removed because `setup_smoke_tests` calls `setup_integration_test` which restores the cypress cache